tests/Rector/MethodCall/GetMockBuilderGetMockToCreateMockRector/Fixture/skip_only_methods_next_call_method.php.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Rector\PHPUnit\Tests\Rector\MethodCall\GetMockBuilderGetMockToCreateMockRector\Fixture;
+
+final class SkipOnlyMethodsNextCallMethod extends \PHPUnit\Framework\TestCase
+{
+    public function testDestructDoesNotThrowException()
+     {
+        $email = $this->getMockBuilder(Email::class)
+            ->disableOriginalConstructor()
+            ->onlyMethods(['sendCommand'])
+            ->getMock();
+         $email->method('sendCommand')
```

Comment:
@samsonasik does this cover the test case where mock creation is in `setUp` while method invocation is in a test method? The fix assumes both mock method creation and invocation are both in the same method body whereas there may be cases where the creation is in `setUp`.

I think another fix (although not yet tested) is to check the array argument supplied to `onlyMethods` if non-empty-array. If empty, it is indeed useless.

Reply:
Those are already deprecated anyway so I'm not sure if there's a need.